vizro-core/src/vizro/models/_components/form/_alert.py

```diff
@@ -25,9 +21,9 @@ class Alert(VizroBaseModel):
     """
 
     type: Literal["alert"] = "alert"
-    text: str = Field(..., description="Text to be displayed in the alert.")
+    text: str = Field(default=..., description="Text to be displayed in the alert.")
```

Comment:
```suggestion
    text: str = Field(description="Text to be displayed in the alert.")
```
The docs says "However, the usage of the [ellipsis](https://docs.python.org/3/library/constants.html#Ellipsis) is discouraged as it doesn't play well with type checkers." https://docs.pydantic.dev/2.10/concepts/models/#required-fields

Reply:
Yes, good shout. I think we should always delete the `...` as you suggest, but when having a default, we should use `default` as a keyword argument, as this could fix an annoyance (https://github.com/python/mypy/issues/14850), although in our case atm it doesnt